src/include/dhcp-server/widgets.rb

```diff
@@ -916,6 +838,39 @@ def AllSettingsButtonHandle(key, event)
       :main
     end
 
+    # Handle function for the 'Apply' button
+    def handle_apply(_key, event)
+      event_id = event["ID"]
+      if event_id == "apply"
```

Comment:
ugh, that really nasty ugly trick :( I worry that some code relie on it. Usually it is not visible as cwm contain data fields that react on change and not action buttons. Maybe that is reason.